string agent_name

float64 agent_pose_x
float64 agent_pose_y
float64 agent_heading
float64 agent_vel_mag
